lib/submit/submitSchema.rb

```diff
@@ -286,7 +288,7 @@ def depositItem(input, replace:)
     out = ssh.exec_sc!("/apps/eschol/subi/lib/subiGuts.rb " +
                  "#{replace == :files ? "--replaceFiles" : replace == :metadata ? "--replaceMetadata" : "--depositItem"} " +
                  "#{shortArk} " +
-                 "'#{actionVerb} at oapolicy.universityofcalifornia.edu' " +
+                 "#{comment} " +
```

Comment:
I wonder why it is hardcoded to oapolicy url. Does this mean it there was an assumption that all deposits using API are from RT2?

Reply:
yeah, this must have been the assumption for some reason.  Martin might remember.